src/createPackage/models/createPackageManager.ts

```diff
@@ -338,6 +343,81 @@ export class CreatePackageManager {
     await fsPromise.writeFile(metadataFilePath, recordMetadata);
   }
 
+  public async createExportJsonMetadata(job: JobExportResponse): Promise<void> {
+    this.logger.info(`Creating metadata.json file for gpkg in path "${job.parameters.relativeDirectoryPath}" for jobId ${job.id}`);
+    const record = await this.rasterCatalogManager.findLayer(job.internalId as string);
+    const featuresRecords = parseFeatureCollection(job.parameters.roi);
+
+    const maxResolutionDeg = Math.max(
+      record.metadata.maxResolutionDeg as number,
+      Math.min(...featuresRecords.map((records) => records.targetResolutionDeg))
+    );
+    const maxResolutionMeter = Math.max(
+      record.metadata.maxResolutionMeter as number,
+      Math.min(...featuresRecords.map((records) => records.targetResolutionMeter))
+    );
+
+    const metadataFileName = job.parameters.fileNamesTemplates.metadataURI;
+    const directoryName = job.parameters.relativeDirectoryPath;
+    const metadataFullPath = concatFsPaths(this.gpkgsLocation, directoryName, metadataFileName);
+    let combinedFootprint = undefined;
+    try {
+      const intersectedFeatures = this.featuresFootprintIntersects(
+        job.parameters.roi.features as Feature<Polygon | MultiPolygon>[],
+        record.metadata.footprint as Polygon | MultiPolygon
+      );
+      const fc: FeatureCollection<Polygon | MultiPolygon> = featureCollection(intersectedFeatures);
+      combinedFootprint = featureCombine(fc).features[0].geometry as unknown as MultiPolygon;
+    } catch (error) {
+      this.logger.error(error, `Failed to match features intersection with footprint`);
+    }
+    record.metadata.footprint = combinedFootprint ? combinedFootprint : record.metadata.footprint;
+    record.metadata.maxResolutionDeg = maxResolutionDeg;
+    record.metadata.maxResolutionMeter = maxResolutionMeter;
+
+    const layerPolygonPartFeatures: Feature[] = [];
+    for (const featureRecord of featuresRecords) {
+      for (const feature of (record.metadata.layerPolygonParts as FeatureCollection).features) {
+        const intersectedFeature = intersect(featureRecord.geometry as Polygon | MultiPolygon, feature.geometry as Polygon | MultiPolygon);
+        if (!intersectedFeature) {
+          continue;
+        }
+        if (feature.properties?.Resolution !== undefined) {
+          const maxResolutionDeg = Math.max(featureRecord.targetResolutionDeg, feature.properties.Resolution as number);
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          intersectedFeature.properties = { ...feature.properties, Resolution: maxResolutionDeg };
+        } else {
+          this.logger.error({ ...feature, msg: `LayerPolygonPart not include property of type 'Resolution` });
```

Comment:
error message should include jobId and other relevant fields

Reply:
added job id, error reason already written in msg, and it print the actual feature to understand how he looks like

Reply:
done